README.md

```diff
@@ -151,6 +151,24 @@ script:
 
 The monitoring dashboard is accessible at `http://kubernetes-master-1.kubernetes-playground.local/monitoring/cluster`
 
+### Kites experiments
+
+#### How to deploy the net-test-dev DaemonSet
+
+On the Master node, run:
+`kubectl apply -f /vagrant/kubernetes/kites/net-test-dev_ds.yaml`
```

Comment:
I'd prefer to minimize copy and paste. Please use an approach similar to the other workloads to deploy on the cluster (see the distributed filesystem for example):

1. Prepare a shell script with the necessary steps.
1. Ask the user to run that script.

Reply:
Going further we should use ansible for everything, but having a shell script ready is a good start.